californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
Yes that would be a next step. And no, I would not introduce this with this PR.

This PR is backwards compatible, only adds proper OSGi meta-data in all modules and does not break anything. The change in californium-osgi would be a breaking change for current consumers, because they suddenly need to add two additional bundles to their runtime.

I therefore suggest to make that change in another PR. Actually I would even introduce an even extended approach to change californium-osgi. But I'm still thinking about how this can be done and included in the build process.